uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -572,7 +576,7 @@ define([
         timeObject.amount = timeObject.maxAmount || '0';
```

Comment:
Do we relally need `timeObject.amount = timeObject.maxAmount **|| '0'**;` ? As for me if 
 `response.number_of_hours.toString()` does not exist `timeObject.maxAmount` will already be "'0'" right?